src/main/java/duke/command/DefaultCommand.java

```diff
@@ -0,0 +1,17 @@
+package duke.command;
+
+/**
+ * Default Command which will get executed when keyword supplied does not match with keywords in CommandMap.
+ */
+public class DefaultCommand implements ICommand {
+    /**
+     * When executed, will print a default statement. Input given will not change
```

Comment:
Maybe move 'Input given will not change' to the same line as 'how function gets executed.' Noticed it in some other places as well